analysis/local_analog_calibration.py

```diff
@@ -21,7 +21,7 @@ def bian18_R23(OH):
     return R23_p(OH)
 
 
-def bian18_O32(O32):
+def bian18_OH(O32):
```

Comment:
This is very confusing as OH determination is possible via R23 and O32. It's not clear which one this is. Please use more explicit name in the function, e.g., `bian18_O32_OH`

Reply:
Ok, I will change the R23 one as well.